mssqlcli/mssqlcliclient.py

```diff
@@ -139,18 +139,14 @@ def execute_single_batch_query(self, query):
         while not query_request.completed():
             query_response = query_request.get_response()
             if query_response:
-                if isinstance(query_response, queryservice.QueryExecuteErrorResponseEvent):
-                    yield self.tabular_results_generator(column_info=None, result_rows=None,
-                                                          query=query, message=query_response.error_message,
-                                                          is_error=True)
-                elif isinstance(query_response, queryservice.QueryMessageEvent):
```

Comment:
We are we deleting this? We need it to hamdle Error response events

Reply:
Ahh ok I see now what happened. Thanks

---
In reply to: [161077826](https://github.com/dbcli/mssql-cli/pull/140#discussion_r161077826) [](ancestors = 161077826)